src/components/circle.js

```diff
@@ -4,82 +4,110 @@ import eventBinder from '../utils/eventsBinder.js'
 import propsBinder from '../utils/propsBinder.js'
 import MapElementMixin from './mapElementMixin';
 import getPropsValuesMixin from '../utils/getPropsValuesMixin.js'
+import generatePropsToBind from "../utils/generatePropsToBind"
 
+const twoWayProps = ["center","radius"]
+const excludedProps = ["bounds"]
 const props = {
-    center: {
-        type: Object,
-        twoWay: true,
-        required: true
-    },
-    radius: {
-        type: Number,
-        default: 1000,
-        twoWay: true
-    },
-    draggable: {
-        type: Boolean,
-        default: false,
-    },
-    editable: {
-        type: Boolean,
-        default: false,
-    },
-    options: {
-        type: Object,
-        twoWay: false
-    }
+  center: {
+    type: Object,
+    required: true
+  },
+  radius: {
+    type: Number,
+    default: 1000,
+  },
+  bounds: {
+    type: Object
+  },
+  draggable: {
+    type: Boolean,
+    default: false,
+  },
+  editable: {
+    type: Boolean,
+    default: false,
+  },
+  options: {
+    type: Object
+  }
 }
 
 const events = [
-    'click',
-    'dblclick',
-    'drag',
-    'dragend',
-    'dragstart',
-    'mousedown',
-    'mousemove',
-    'mouseout',
-    'mouseover',
-    'mouseup',
-    'rightclick'
+  'click',
+  'dblclick',
+  'drag',
+  'dragend',
+  'dragstart',
+  'mousedown',
+  'mousemove',
+  'mouseout',
+  'mouseover',
+  'mouseup',
+  'rightclick'
 ]
 
 export default {
-    mixins: [MapElementMixin, getPropsValuesMixin],
-    props: props,
-    version: 2,
-
-    render() { return '' },
+  mixins: [MapElementMixin, getPropsValuesMixin],
+  render(h) {
+    return h( // So that infowindows can have a marker parent
```

Comment:
Can the info window be bound to a circle? Quoting the reference API: "In the core API, the only anchor is the Marker class."